ga4gh/server/datamodel/variants.py

```diff
@@ -1040,6 +1057,33 @@ class HtslibVariantAnnotationSet(AbstractVariantAnnotationSet):
     Class representing a single variant annotation derived from an
     annotated variant set.
     """
+    CSQ_FIELDS = ("alternate_bases", "gene", "feature_id",
```

Comment:
I aspire to have these tuples as part of the registry editor.

It should be possible to easily write a VCF->ANN plugin that accepts a tuple or dictionary that can effectively parse a ANN field.